articles/openshift/tutorial-create-cluster.md

```diff
@@ -83,6 +83,14 @@ When running the `az aro create` command, you can reference your pull secret usi
 
 If you are copying your pull secret or referencing it in other scripts, your pull secret should be formatted as a valid JSON string.
 
+### Prepare a custom domain for your cluster (optional)
+
+If you provide a custom domain for your cluster, for example, **foo.example.com**, note the following points:
+
+* The OpenShift console will be available at a URL such as `https://console-openshift-console.apps.foo.example.com`, instead of the built-in domain `https://console-openshift-console.apps.<random>.<location>.aroapp.io`.
+
+* By default, OpenShift uses self-signed certificates for all of the routes created on `*.apps.<random>.<location>.aroapp.io`.  If you choose to use custom DNS after connecting to the cluster, you will need to follow the OpenShift documentation to [configure a custom CA for your ingress controller](https://docs.openshift.com/container-platform/4.3/authentication/certificates/replacing-default-ingress-certificate.html) and a [custom CA for your API server](https://docs.openshift.com/container-platform/4.3/authentication/certificates/api-server.html).
+
```

Comment:
We need to also document that they must create 2 DNS A records in their DNS server for the --domain specified:

- "api" - pointing to the api server
- "*.apps" - pointing to the ingress

Reply:
@klamenzo can you make these changes, and I'll pull this into MSFT docs?

Reply:
@ms-jasondel I updated this (verbatim, thank you!)
My first reaction is that we should add some guidance for the customer on where to get the value of the location of their api server and ingress.

Do you know how someone can get these values for their cluster?

Reply:
I don't have a cluster up at the moment, but these values are returned from az aro show.  With a --query argument we can create a query that returns both of these IP addresses.  I'll try to connect to someone else's cluster to construct the query.

Reply:
This command will get you the IPs:

az aro show -n <name> -g <resource-group> --query '{api:apiserverProfile.ip, ingress:ingressProfiles[0].ip}'

Reply:
Got it. Thanks for that command.

I also called out that you make these change after creating the cluster.